lib/active_record/mysql/enum/mysql_adapter.rb

```diff
@@ -45,20 +45,30 @@ def type_to_sql(type, limit: nil, **_options) # :nodoc:
           end
         end
 
+        if Gem::Version.new(Rails.version.to_f.to_s) < Gem::Version.new('7.0')
```

Comment:
`Rails.version` already outputs a string, so the `to_f.to_s` is unnecessary here, would you mind removing this?

Reply:
Sorry about this - I was testing this code back when it was Rails 7.0.0.alpha1 and it was breaking my check here:

```ruby
Gem::Version.new("7.0.0.alpha1") < Gem::Version.new('7.0')
=> true
```

so I was forcing it to a single decimal digit and then back to a string for Gem::Version.

```
irb(main):008:0> "7.0.0.alpha1".to_f.to_s
=> "7.0"
irb(main):009:0> Gem::Version.new("7.0.0.alpha1".to_f.to_s) < Gem::Version.new('7.0')
=> false
```

Anyway, not necessary now and should have removed this before opening. I have removed it now.